makefile

```diff
@@ -2,31 +2,56 @@ help:
+	@echo "	Setup RDS - mysql, documentdb:		make setup-dbs-rds"
+	@echo "	Setup rabbitmq-operator:			make setup-rabbitmq-operator"
+	@echo "	Deploy application via:			make setup-app"
 	@echo "	Setup Ingress gateway:		make setup-gateway"
-	@echo "	Setup kiali and jaeger via:		make setup-addons"
-	@echo "	Setup Litmus-3 chaos tool via:		make setup-litmus"
-	@echo "	Setup APM via:				make setup-apm"
+	@echo "	Setup kiali and jaeger via:		make setup-istio-observability-addons"
 	@echo ""
 	@echo "Command can be used for cleanup:"
-	@echo "	Clenaup all via:			make cleanup-cluster"
+	@echo "	Clenaup cluster via:			make cleanup-cluster"
+	@echo "	Clenaup all via:			make cleanup"
 
-setup: start-cluster install-asg setup-observability setup-istio deploy-app setup-gateway setup-addons
+setup: setup-cluster setup-cluster-autoscaler setup-istio setup-observability setup-istio-observability-addons setup-dbs-rds setup-rabbitmq-operator setup-app setup-gateway setup-keda
 
-start-cluster:
+cleanup: destroy-istio-gateway destroy-dbs-rds cleanup-cluster
+
+setup-cluster:
 	eksctl create cluster -f infra/eksctl.yaml
 
+setup-cluster-autoscaler:
+	eksctl utils associate-iam-oidc-provider \
+	--region=us-east-1 --cluster prod-eks-cluster \
+	--approve
+	aws iam create-policy  \
+	--policy-name k8s-asg-policy \
+	--policy-document file://./infra/asg-policy.json
+	eksctl create iamserviceaccount \
+	--region=us-east-1 --name cluster-autoscaler \
+	--namespace kube-system \
+	--cluster prod-eks-cluster \
+	--attach-policy-arn "arn:aws:iam::813864300626:policy/k8s-asg-policy" \
+	--approve \
+	--override-existing-serviceaccounts
+	kubectl apply -f infra/cluster-autoscale.yaml
+	kubectl -n kube-system \
+	annotate deployment.apps/cluster-autoscaler \
+	cluster-autoscaler.kubernetes.io/safe-to-evict="false"
+
+setup-istio:
+	helm repo add istio https://istio-release.storage.googleapis.com/charts && helm repo update
+	helm upgrade --install istio-base istio/base -n istio-system --create-namespace --version 1.17.2 --wait --timeout 2m0s
+	helm upgrade --install istiod istio/istiod -n istio-system --version 1.17.2 --set meshConfig.defaultConfig.tracing.zipkin.address=zipkin.monitoring:9411 --set pilot.traceSampling=100 --wait --timeout 2m0s
+	helm upgrade --install istio-ingressgateway istio/gateway -n istio-system --version 1.17.2 --wait --timeout 2m0s
+
 setup-observability:
-	make setup-istio
 	helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
 	helm repo add grafana https://grafana.github.io/helm-charts
 	helm repo update
 	helm upgrade --install prometheus-stack prometheus-community/kube-prometheus-stack --values ./monitoring/chart-values/prometheus-values.yaml -n monitoring --create-namespace --version 52.0.0
-	kubectl apply -f ./monitoring/istio-addons/prometheus-vs.yaml
-	kubectl apply -f ./monitoring/istio-addons/grafana-vs.yaml
 	kubectl apply -f ./monitoring/dashboard.yaml
 	kubectl apply -f ./monitoring/dashboard-rabbitmq.yaml
```

Comment:
add `kubectl apply -f ./monitoring/dashboard-rds.yaml` for installing rds dashboard